docs/tab-widgets/tab-widget-code/code.asciidoc

```diff
@@ -0,0 +1,166 @@
+// Defining styles and script here for simplicity.
+++++
```

Comment:
ToDo: Figure out which tab should be the default selection, and set it.

Reply:
I don't think we need to include this anymore since as of elastic/docs#2180, this is included in the doc build itself and doesn't need to be in each repo that uses the tab widgets.

Reply:
NOTED for future action.  Thanks for the info. While I'm still developing content, it's handy to be able to fire off local builds for quickie testing. 

I'll remove the file before I finalize this PR.

Reply:
Sounds great, let me know if I can help at all.